GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/PrefetchVerbTests.cs

```diff
@@ -88,9 +88,9 @@ public void PrefetchFolderListFromFile()
                 {
                     "# A comment",
                     " ",
-                    "gvfs/",
-                    "gvfs/gvfs",
-                    "gvfs/"
+                    "GVFS/",
+                    "GVFS/GVFS",
+                    "GVFS/"

```

Comment:
@derrickstolee @kewillford @chrisd8088 I'm thinking we might want to parameterize test data like this so that on case sensitive file system we use matching case (e.g. "GVFS") and on case insensitive file systems we use both cases ("gvfs" and "GVFS").

With the current set of changes we're losing validation that case-insensitive path matching works as expected on Mac and Windows.

Thoughts?

(This comment applies to all of the tests whose file paths were updated to match what's in the repo)

Reply:
I think I've updated this PR with all of your suggestions below, @wilbaker -- the exception is this one.  I could also just add some `FileSystemHelper.CaseSensitiveFileSystem ? ... : ...` code here as well, if you think that's better.  Let me know, or I'll try to tackle it early tomorrow, if time permits, and otherwise in a week or so.

Thanks again for all the review; much appreciated!

Reply:
> I could also just add some FileSystemHelper.CaseSensitiveFileSystem ? ... : ... code here as well, if you think that's better.

I think that would work well.  

If you could leave some of these entries lower-case on case-insensitive file systems then I think this PR is ready to go.

Reply:
OK -- as we discussed just now, I'll put those changes into a followup PR next week, and maybe try to track down the source of the uppercase SHA1s, if I can, in another.  But I'm going to go ahead and merge this now -- thank you for all your support!